cmake/LLVM.cmake

```diff
@@ -783,3 +783,45 @@ execute_process(COMMAND "${CLANG}" "--print-resource-dir" OUTPUT_VARIABLE RESOUR
 string(STRIP "${RESOURCE_DIR}" RESOURCE_DIR)
 set(CLANG_RESOURCE_DIR "${RESOURCE_DIR}" CACHE INTERNAL "Clang resource dir")
 
+##########################################################
+
+if(LLVM_VERSION_MAJOR GREATER 17)
+  # these are experimental options and may change in future
```

Comment:
What prevents switching on by default?

Reply:
i'll have to add libmvec detection, otherwise nothing

Reply:
OK please do. It's always better for rot-avoidance if we can enable by default, especially this useful feature.